### PR TITLE
feat(remix): Add OTEL auto-instrumentation instructions.

### PR DESCRIPTION
**Pull request by onurtemizkan**
Documents the changes in https://github.com/getsentry/sentry-javascript/pull/12110

### CODE REVIEW

File: docs/platforms/javascript/guides/remix/manual-setup.mdx [hunk 1]
```diff
@@ -209,24 +237,18 @@ To enable readable stack traces, <PlatformLink to="/sourcemaps">configure source
...
 
-const createSentryRequestHandler =
-  wrapExpressCreateRequestHandler(createRequestHandler);
+const app = express();
 
-app.all("*", createSentryRequestHandler(/* ... */));
+// ...
 ```
 
 ### Usage with Vite development mode (only for SDK versions < 7.106.0)
```
Comment (AbhiPrasad) [inline docs/platforms/javascript/guides/remix/manual-setup.mdx:254]: Do these docs still apply?

**Review by AbhiPrasad — COMMENTED** [review]
After we merge this in, can we update the docs so that we don't have a separate remix v2 section anymore?

We can add a remix v1 section instead given everyone with new projects should be on remix v2.

File: docs/platforms/javascript/guides/remix/manual-setup.mdx (hunk 1 above)
Comment (onurtemizkan) [reply docs/platforms/javascript/guides/remix/manual-setup.mdx:254]: That version doesn't feel very relevant anymore. Should I delete the section?

Comment (AbhiPrasad) [reply docs/platforms/javascript/guides/remix/manual-setup.mdx:251]: Let's remove it!

**Review by vivianyentran — APPROVED** [review]
Added a suggestion. Thanks for updating!

**Pull request merged**